### ISSUE TITLE
"go.vetFlags" fails to vet code in 0.6.62

### ISSUE THREAD

**Issue by seanpfeifer** [Microsoft/vscode-go#1073]
The change in #1023  (released in 0.6.62) to use "go vet" instead of "go tool vet" has broken the ability to use "go.vetFlags".

"go vet" fails silently in the Output tab (and not code is actually vetted) if "go.vetFlags" are defined as per the example for the flag:
`"go.vetFlags": ["-all", "-shadow"]`

Per the "go vet" command:
`To run the vet tool with specific options, run 'go tool vet'.`

**Comment by seanpfeifer** [comment]
Simple test case:
```go
package main

import "fmt"

func main() {
	fmt.Printf("%s")
}
```

Expected:
`missing argument for Printf("%s"): format reads arg 1, have only 0 args`

Actual when assigning a value to "go.vetFlags", such as `"go.vetFlags": ["-all", "-shadow"]`:
No vet information reported.

**Comment by ramya-rao-a** [comment]
Thanks for catching this @seanpfeifer!

Fixed and also added a unit test so that the tests can catch it if broken again.

Fix will be out in the next update

**Comment by ramya-rao-a** [comment]
The fix is now out in the latest update (0.6.63) to the Go extension
